djedi/plugins/list.py

```diff
@@ -0,0 +1,162 @@
+from .base import DjediPlugin
+from cio.plugins import plugins
+from cio.node import Node
+import cio
+import json
+
+
+class ListPlugin(DjediPlugin):
+    ext = 'list'
+
+    def get_editor_context(self, **kwargs):
+        uri = kwargs['uri']
+        plugin_ext = self.get_query_param(uri, 'plugin')
+        if plugin_ext:
+            kwargs['plugin'] = plugin_ext
```

Comment:
We probably need to `get_editor_context` of the child plugin here and update kwargs dict with it

Something like:

```
        if plugin_ext:
            kwargs['plugin'] = plugin_ext

            kwargs.update(
                plugins.get(plugin_ext).get_editor_context(**kwargs)
            )
```

Reply:
Hmm yes, you're right. Any idea how we can test it?

Reply:
We can maybe use the existing image plugin in the test case (since it returns some extra variables in its editor context, like it seems you've already done), or, to have the test case self contained, write a small custom plugin inline in testcase and use that instead?

Reply:
Yep! I added a testcase verifying the context data for the [image plugin](https://github.com/5monkeys/djedi-cms/pull/61/commits/ef5191e8501a840e5f05dd8d0f071011e0ad437a#diff-7935b6aed823a03415a6c902fbfd62d3R532)